src/scripts/createLocationAutocompleteData/index.ts

```diff
@@ -0,0 +1,345 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * The data that is being downloaded in this script is provided by
+ * GeoNames (https://www.geonames.org/). We are using the data to compile a set
+ * of US cities and states that match the needs of this project. Their work is
+ * licensed under a Creative Commons Attribution 4.0 License:
+ * https://creativecommons.org/licenses/by/4.0/.
+ *
+ * All database dumps and table definitions can be found here:
+ * https://download.geonames.org/export/dump/.
+ */
+
+import https from "https";
+import {
+  createWriteStream,
+  existsSync,
+  mkdirSync,
+  readFileSync,
+  rmSync,
+  writeFileSync,
+} from "fs";
+import { dirname, resolve } from "path";
+import { fileURLToPath } from "url";
+import AdmZip from "adm-zip";
+
+import {
+  TAlternateNameData,
+  TLocationData,
+  IRelevantLocation,
+  IRelevantLocationAlternate,
+  IDataFileUrls,
+} from "./types.d";
+
+const REMOTE_DATA_URL = "https://download.geonames.org/export/dump";
+const DATA_COUNTRY_CODE = "US";
+const LOCATIONS_DATA_FILE = "locationAutocompleteData.json";
+const FETCH_REMOTE_DATASETS = true;
+const CLEANUP_TMP_DATA_AFTER_FINISHED = true;
+
+// Only include populated place a city, town, village, or other
+// agglomeration of buildings where people live and work.
```

Comment:
I think a word or character (`:`?) is missing here - I don't quite know what this is supposed to say :sweat_smile: 

(Alternatively, I should drink some coffee.)